android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java

```diff
@@ -1252,13 +1252,7 @@ public void onUpdated() {
             wallpaperRetrievedCallback = new FetchWallpaperWorkerTask.WallpaperRetrievedCallback() {
         @Override
         public void bgWallpaperRetrieved(Bitmap bgWallpaper) {
-            if (ChromeFeatureList.isEnabled(BraveFeatureList.BRAVE_NEWS)) {
-                if (BraveActivity.getBraveActivity() != null && mTabProvider.get() != null
-                        && !mTabProvider.get().isIncognito()) {
```

Comment:
I believe we still should check for private tab as it's not supposed to have background image.

Reply:
In case of private tab, I think we are using different layout.

Reply:
Hm, I wonder why we had that check before. Ok, just double check private tab in not affected by this change.